openmdao/visualization/n2_viewer/src/N2Style.js

```diff
@@ -67,7 +67,7 @@ class N2Style {
             'g.subsystem > rect': {
                 'cursor': 'pointer',
                 'fill-opacity': '.8',
-                'fill': N2Style.color.group
+                'fill': '#6092B5',
```

Comment:
why is this color value hard coded now rather than using the `N2style.color.group`...